okhttp/src/commonJvmAndroid/kotlin/okhttp3/Cache.kt

```diff
@@ -609,6 +633,20 @@ class Cache internal constructor(
             .writeByte('\n'.code)
         }
 
+        if (requestMethod == "QUERY") {
+          // Write the body of a QUERY request.
```

Comment:
Do we need to store the request body in cache? 

Looking at the code, it looks like storing a request in the cache is already destructive as only the `vary` headers of the request are stored (not all of them). 

I think I wouldn't mind too much if `Cache.Entry.body` was missing and saving some bytes there.

Reply:
True, it just needs to be part of the key. I think.

Reply:
If we can avoid changing the format, that would be good.

Reply:
Reverted